pom.xml

```diff
@@ -592,7 +598,7 @@
 			</plugin>
 
 			<!-- Scala Code Style, most of the configuration done via plugin management -->
-			<plugin>
```

Comment:
could you please explain why you changed this file?

Reply:
@VenturaDelMonte, sorry for disable scala code style plugin. It was a mistake. Now it is enabled. It is only necessary change pom.xml to add a new dependency (scalacheck)